src/SDKs/Azure.Base/data-plane/Azure.Base/Response.cs

```diff
@@ -59,7 +61,7 @@ public bool TryGetHeader(string name, out string value)
             return TryGetHeader(utf8Name, out value);
         }
 
-        public void Dispose() => _message.Dispose();
+        public void Dispose() => _httpResponse.Dispose();
```

Comment:
Unless I'm misreading, it seems that `_httpResponse` could be `null`.   Should we consider using `?.Dispose()` to ensure that `Dispose` doesn't throw?